source/assets/stylesheets/components/_cards.scss

```diff
@@ -0,0 +1,156 @@
+// Variables
+
+$light:  #febbbb;
+$medium: #fe9090;
+$dark:   #ff5c5c;
+$dark-v1: #c84848;
+$text-color: #999;
+$cta: #00c5a7;
+$card-font-small: 15px;
+
+
+// Global Styles
+/*#press {
+  background: $light;
+  background: linear-gradient(135deg, $light 0%,$medium 45%,$dark 100%);
+  font-size: 10px;
+  padding: 2em;
+}
+.full-screen {
+  height: 100vw;
+  position: relative;
+}*/
+
+
+// General content
+.title {
+  text-align: center;
```

Comment:
This is too generic to be un-scoped. It has not been decided yet what will be the convention for naming but I'm personnaly using BEM convention for years and its has been proven efficient and widely used.